common4j/src/main/com/microsoft/identity/common/java/authorities/Authority.java

```diff
@@ -344,6 +351,10 @@ public static boolean isKnownAuthority(Authority authority) {
             return false;
         }
 
+        if (BuildConfig.ALLOW_ONEBOX_AUTHORITIES) {
+            return true; // onebox authorities are always considered to be known.
```

Comment:
>return true [](http://example.com/codeflow?start=12&length=11)

Shouldn't you still check if input authority is OneBox? 

Even if the build flag is enabled, you can use other public authorities and MSAL configs.

Reply:
Sure. 

Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2559/commits/58d3c2fb0877dc1854096892d267d30d477f7969